### ISSUE TITLE
iOS 18.1.1 support

### ISSUE THREAD

**Issue by Splash04** [leminlimez/Nugget#220]
Do Nugget work with 18.1.1?

**Comment by Mario0051** [comment]
Mobile Gestalt still works, though not too sure on the rest.

**Comment by answngud04** [comment]
Yes, Nugget 4.1 works on Windows. I tested it with an iPhone 14 Pro, including features like Dynamic Island, Apple Intelligence, SOS, AOD and silent camera.

**Comment by aygz** [comment]
Yes, Nugget 4.1  Windows version worked  in IOS 18.1.1. I enabled AOD in iPhone 16 , but tap to wake still doesn't work. If this is solved, it will be perfect

**Comment by mehrdadrn** [comment]
> Yes, Nugget 4.1 Windows version worked in IOS 18.1.1. I enabled AOD in iPhone 16 , but tap to wake still doesn't work. If this is solved, it will be perfect

why do you need tap to wake on iphone 16!!!????

**Comment by A350-900** [comment]
> > Yes, Nugget 4.1 Windows version worked in IOS 18.1.1. I enabled AOD in iPhone 16 , but tap to wake still doesn't work. If this is solved, it will be perfect
> 
> why do you need tap to wake on iphone 16!!!????

how about you try enabling AOD and see if tap to wake works!!!????

**Comment by lslqtz** [comment]
Works for me on nugget 4.1 but not on 4.2. (iPad Pro 6th gen)
